mitosheet/mitosheet/tests/test_utils.py

```diff
@@ -749,15 +749,32 @@ def generate_graph(
                 'params': {
                     'graph_id': graph_id,
                     'graph_preprocessing': {
-                    'safety_filter_turned_on_by_user': safety_filter_turned_on_by_user
+                        'safety_filter_turned_on_by_user': safety_filter_turned_on_by_user
                     },
                     'graph_creation': {
                         'graph_type': graph_type,
                         'sheet_index': sheet_index,
                         'x_axis_column_ids': xAxisColumnIDs,
                         'y_axis_column_ids': yAxisColumnIDs,
                     },
-                    'graph_styling': {},
+                    'graph_styling': {
+                        'title': {
+                            'title': None,
+                            'visible': True
+                        },
+                        'xaxis': {
+                            'title': None,
+                            'visible': True,
+                            'rangeslider': {
+                                'visible': True
+                            }
+                        },
+                        'yaxis': {
+                            'title': None,
+                            'visible': True
+                        },
+                        'showlegend': True
+                    },
```

Comment:
Can we make these optional arguments to the call? And in turn, can you test that they work (e.g. you can set them without error, and that the are in the params of the resulting graph?